helpers/simg2img.mk

```diff
@@ -2,7 +2,7 @@
 
 SIMG2IMG_SOURCES?= nosourcessupplied
 
-CXXFLAGS+= -std=c++0x
+CXXFLAGS+= -std=c++11
```

Comment:
We should unify flags to either both gnu+11 or both c++11.

Reply:
Then it should be gnu++11 as we use typeof.